package.json

```diff
@@ -88,8 +88,8 @@
     "uglify-js": "^2.6.1"
   },
   "peerDependencies": {
-    "react": "^0.14.0",
-    "react-dom": "^0.14.0"
```

Comment:
How about `0.14.x - 15.x` , since otherwise this breaks compatibility with 0.14.